docs/odk-workflows/RepositoryFileStructure.md

```diff
@@ -30,9 +30,9 @@ These are the components in MONDO-INGEST
 
 | Filename | URL |
 | -------- | --- |
+| doid.owl | http://purl.obolibrary.org/obo/doid.owl |
```

Comment:
@matentzn These are the changes related to my updates to `mondo-ingest-odk.yaml`. I just sorted the list of ontologies alphabetically. This made similar ordering changes as well to the `Makefile`.